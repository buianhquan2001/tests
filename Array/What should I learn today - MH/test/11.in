20
9
12 10 22 23 15 20 16 18 3 
9
8 3 9 9 6 1 7 15 10 
18
8 10 14 6 23 3 3 5 21 25 24 16 21 14 19 19 3 3 
26
25 4 13 19 24 5 15 7 15 25 5 2 15 11 13 9 13 4 20 9 22 25 7 17 1 21 
19
25 14 8 17 19 11 3 2 2 25 25 22 7 8 11 25 1 21 10 
4
12 1 5 14 
20
14 17 8 4 9 13 23 11 11 11 2 3 3 8 14 1 22 6 12 5 
21
13 13 11 10 8 11 8 12 1 22 4 4 6 8 18 22 4 22 2 21 21 
8
14 17 17 15 16 15 2 6 
8
25 18 22 14 12 16 16 23 
25
2 20 23 24 14 22 19 9 22 16 4 11 25 23 18 17 1 2 3 9 22 2 12 19 17 
3
4 19 6 
13
23 2 9 11 11 11 2 8 7 18 19 1 14 
5
20 8 18 16 17 
24
14 25 24 16 4 24 18 19 2 17 13 19 22 16 2 25 14 11 12 14 11 6 12 10 
10
6 16 15 15 8 14 14 16 9 14 
27
18 23 17 18 15 11 17 6 16 1 18 22 6 14 4 23 18 21 8 5 13 12 9 5 5 8 24 
30
25 13 11 6 5 6 17 24 2 24 10 14 8 12 11 3 22 1 2 20 15 5 13 3 6 7 3 25 1 19 
19
25 17 13 19 1 2 23 13 13 5 24 12 12 7 11 21 22 22 11 
1
24 
