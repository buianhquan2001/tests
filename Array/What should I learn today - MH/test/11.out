math
math
math
math
math
math
english
physics
english
math
physics
english
math
math
math
math
physics
english
math
math
